src/commands/cmd_json.cc

```diff
@@ -52,7 +52,33 @@ class CommandJsonGet : public Commander {
   }
 };
 
+class CommandJsonStrAppend : public Commander {
+public:
+    Status Execute(Server *svr, Connection *conn, std::string *output) override {
+      redis::Json json(svr->storage, conn->GetNamespace());
+
+      JsonValue result;
+      std::vector<std::string> paths;
+      std::vector<std::string> values;
```

Comment:
Seems that `path` and `value` should be `std::string`?

You can also take a look at `CommandParser`.

```
std::string path_or_root = "$";
std::string value;
if (args_.size() == 3) {
  value = ...
} else {
  path = 
  value = 
}
```